sbin/fix_soundcard_mixer_ctrls.py

```diff
@@ -21,40 +21,40 @@
 # GNU General Public License for more details.
 #
 # For a full copy of the GNU General Public License see the LICENSE.txt file.
-# 
-#********************************************************************
+#
+# ********************************************************************
 
+import zynconf
```

Comment:
zynconf after os & sys?

Reply:
Why is the import order important? Please describe the reasoning behind this suggested change.

Reply:
Pep-8 specified an order https://www.python.org/dev/peps/pep-0008/#id23